packages/react/focus-scope/src/FocusScope.stories.tsx

```diff
@@ -32,7 +32,11 @@ export const Basic = () => {
               <input type="text" placeholder="First name" />
               <input type="text" placeholder="Last name" />
               <input type="number" placeholder="Age" />
-              <button type="button" onClick={() => setTrapped(false)}>
+              <button
```

Comment:
Good for testing this PR but do you think we need this since we cover it in the actual test case?

Reply:
oops, left over, nice catch!